src/components/Pagination/Nodes/NavListItemButton.tsx

```diff
@@ -34,7 +34,10 @@ export const NavListItemButton = ({
           "neo-btn-square",
           `neo-btn-square-${variant} neo-btn-square-${variant}--info`,
         )}
-        onClick={(e) => onPageChange(e, pageToNavigateTo)}
+        onClick={(e)=>{
+          if(!isCurrentPage)
+           onPageChange(e, pageToNavigateTo)
+        }}
```

Comment:
This is wrong. The `button` should be disabled and the event shouldn't be propagating up to here.

Reply:
Matthew doesnt want the button to be disabled because the visual changes of the disabled button are too confusing to the user

Reply:
We decided on a few CSS changes that change the active and hover state of the current page button to make it less confusing for the user and this change is here so we arent reloading the page if we click the current page button. otherwise i believe it works all the same. The only thing that was added was the if statement.

Reply:
I think that you should be using the `disabled` attribute and updating the disabled CSS styling. Let's discuss in parking lot. 

If we do decide to go with this solution, you should always wrap your `if`s in curly braces.
```suggestion
        onClick={(e)=>{
          if(!isCurrentPage) {
           onPageChange(e, pageToNavigateTo);
          }
        }}
```